specification/src/chapter5.adoc

```diff
@@ -2,6 +2,72 @@
 
 == Cryptography
 
-<use models enabled by crypto ISA extensions>
+RISC-V supports a number of ISA-level extensions aimed at improving performance for cryptographic operations (scalar and vector). They also include an ISA-level entropy source, and guidelines for data independent execution latency.
 
-<crypto guidelines>
\ No newline at end of file
+See xref:chapter3.adoc#_cryptography[cryptography] +
+See https://github.com/riscv/riscv-crypto
+
+Current ISA level cryptographic extensions work at round level. With the data independent execution latency properties, they can provide some mitigation against some side-channel attacks, such as cache timing attacks. They may not defend fully against some differential power analysis, for example.
+
+Work is on-going to define ISA-level _high assurance cryptography (HAC)_. This work includes defining full-round operations to increase side-channel resistance; adding operations supporting _post-quantum cryptography (PQC)_; and adding ISA-level privilege-based key management.
+
+Cryptographic requirements depend on target ecosystem, as well as on varying regulatory requirements in different geographic regions. This chapter summarizes commonly used cryptographic guidance for secure systems, provided as guidance for development of RISC-V specifications and RISC-V based secure systems.
+
+=== PQC readiness
+
+Quantum safe cryptography is an evolving area of research. For example, see:  https://csrc.nist.gov/projects/post-quantum-cryptography.
+
+NIST has already standardized three PQ algorithms: Kyber (FIPS-203), Dilithium (FIPS-204), and SPHINCS+ (FIPS-205). Kyber is used for general encryption, and Dilihtium and SPHINCS+ are used for digital signatures.
+
```

Comment:
We should use NIST approved names. CRYSTALS-Kyber was input there are differences between FIPS-203 and the the Kyber submission.
Suggest:
ML-KEM (FIPS-203), ML-DSA (FIPS-204), and SLH-DSA (FIPS-205). ML-KEM defines a key-encapsulation mechanism used to establish a shared secret key over a public channel. ML-DSA and SLH-DSA defined digital signature schemes.

Reply:
Done.